### DISCUSSION TITLE
Spatial region

### DISCUSSION THREAD

**Discussion by jaywal04** [spacetelescope/jdaviz#1811] [Q&A] [answered]
I have a question regarding Cubeviz. When I open Cubeviz and I locate the “Line Analysis” tab, I only see an option for "Spectral region” but not “Spatial region.” I was wondering where I can locate “Spatial region” because I have seen others having that option along with "Spatial region.” Any help is appreciated, thanks.

**Comment by kecnry** [comment, marked as answer]
What version of jdaviz do you have?  This capability was introduced in #1583 in September, so is only present in jdaviz 3.0+.

**Reply by jaywal04** [reply]
I just updated jdaviz and "Spatial region" is now showing. I didn't realize I was running the older version of jdaviz. Thank you!
